File: examples/c/env_vars.c
#include <stdio.h>
#include <stdlib.h>

int main(void)
{
    const char *my_var = getenv("MY_VAR");
    const char *debug = getenv("DEBUG");
    const char *greeting = getenv("GREETING");
    printf("MY_VAR=%s\n", my_var ? my_var : "");
    printf("DEBUG=%s\n", debug ? debug : "");
    printf("GREETING=%s\n", greeting ? greeting : "");
    return 0;
}

File: examples/go/env_vars.go
package main

import (
	"fmt"
	"os"
)

func main() {
	fmt.Printf("MY_VAR=%s\n", os.Getenv("MY_VAR"))
	fmt.Printf("DEBUG=%s\n", os.Getenv("DEBUG"))
	fmt.Printf("GREETING=%s\n", os.Getenv("GREETING"))
}
